core/src/main/java/io/questdb/griffin/engine/functions/lt/LtDoubleVVFunctionFactory.java

```diff
@@ -62,9 +63,16 @@ public FuncVV(Function left, Function right) {
 
         @Override
         public boolean getBool(Record rec) {
+            double l = left.getDouble(rec);
+            double r = right.getDouble(rec);
+
+            if (l != l || r != r) {
+                return false;
```

Comment:
We usually return true on `NULL = NULL`. But it's beyond this PR, I created an issue #4423 to make it consistent